Tuist/Interfaces/UIKit/Sources/Data/NetworkAPI/Core/NetworkAPIProtocol.swift

```diff
@@ -17,28 +18,29 @@ extension NetworkAPIProtocol {
         session: Session,
         configuration: RequestConfiguration,
         decoder: JSONDecoder
-    ) -> Single<T> {
-        return session.rx.request(
-            configuration.method,
-            configuration.url,
-            parameters: configuration.parameters,
-            encoding: configuration.encoding,
-            headers: configuration.headers,
-            interceptor: configuration.interceptor
-        )
-        .responseData()
-        .flatMap { _, data -> Observable<T> in
-            Observable.create { observer in
-                do {
-                    let decodable = try decoder.decode(T.self, from: data)
-                    observer.on(.next(decodable))
-                } catch {
-                    observer.on(.error(error))
+    ) async throws -> T {
+        try await withCheckedThrowingContinuation { continuation in
```

Comment:
Not sure did you check about this yet: https://github.com/Alamofire/Alamofire/blob/master/Documentation/AdvancedUsage.md#using-alamofire-with-swift-concurrency

We can use async await directly

Reply:
@suho Good point, I updated to use them directly at https://github.com/nimblehq/ios-templates/pull/525/commits/034edaea3ac6fdc2e0990fa29d7a62a9622525a4, also move that `Data` folder to shared  `Project/Sources/Data`🙏  cc: @blyscuit